src/theme/components/ThemeToggle.tsx

```diff
@@ -29,10 +29,16 @@ const themeModeAtom = atomWithStorage<ThemeMode>('interface_color_theme', ThemeM
 export function SystemThemeUpdater() {
   const setSystemTheme = useUpdateAtom(systemThemeAtom)
 
+  const listener = (event: MediaQueryListEvent) => {
+    setSystemTheme(event.matches ? ThemeMode.DARK : ThemeMode.LIGHT)
+  }
+
   useEffect(() => {
-    DARKMODE_MEDIA_QUERY.addEventListener('change', (event) => {
-      setSystemTheme(event.matches ? ThemeMode.DARK : ThemeMode.LIGHT)
-    })
+    try {
+      DARKMODE_MEDIA_QUERY.addEventListener('change', listener)
```

Comment:
This should include a comment linking to documentation of why the try-catch is required. Does MDN have a compatibility table?
_What_ is unsupported? `addEventListener` _should_ be supported - is it just the `change` event?

Reply:
`addListener` is what's supported on older browsers on `matchMedia`. we could either fallback like this, or we could just not worry about this for older browsers (but not fail). lmk what you think?

Reply:
I think you should fallback if it's just one line.
Either way, could you add an in-code comment explaining why this is necessary?